Cue/app/common/CueColors.js

```diff
@@ -5,6 +5,7 @@
 module.exports = {
   primaryAccent: '#51A7F9',
 
+  primaryTintEvenLighter: '#BED1EA',
```

Comment:
`primaryTintLighter`
`primaryTintOnlyALittleBitLighter`
`primaryTint`

[Maybe we should have gone with numbers instead](https://material.io/guidelines/style/color.html#color-color-palette).